### PR TITLE
docs(cli): add set-version to newdocs

### PR DESCRIPTION
**Pull request by qbzzt**

### CODE REVIEW

File: next-docs/pages/cli/set-version.mdx [hunk 1]
```diff
@@ -1 +1,24 @@
 # mud set-version
+
+Set MUD version in all `package.json` files and optionally backup the previously installed version
+
+## Sample usage
+
+To set the version to the latest `next` release.
+
+```sh copy
+pnpm mud set-version -t next
```
Comment (holic) [inline next-docs/pages/cli/set-version.mdx:10]:
```suggestion
pnpm mud set-version --tag next
```

File: next-docs/pages/cli/set-version.mdx [hunk 2]
```diff
@@ -1 +1,24 @@
 # mud set-version
+
+Set MUD version in all `package.json` files and optionally backup the previously installed version
```
Comment (holic) [inline next-docs/pages/cli/set-version.mdx:3]: we no longer do backups in this way

```suggestion
Set MUD dependency version for all packages in the current workspace.
```

**Review by holic — COMMENTED** [review]
might not be a bad idea to copy some of these docs over to the README too!

**Comment by qbzzt** [comment]
> might not be a bad idea to copy some of these docs over to the README too!

Either that, or add links to them when they are published (which will make sure we don't get out of sync)

File: next-docs/pages/cli/set-version.mdx [hunk 3]
```diff
@@ -1 +1,24 @@
...
+## Options
+
+These are the command line options you can specify on `mud set-version`:
+
+| Option         | Meaning                                                      | Type    | Default value |
+| -------------- | ------------------------------------------------------------ | ------- | ------------- |
+| `--version`    | Show version number                                          | boolean | false         |
+| `--mudVersion` | Set MUD to the given version                                 | string  | true          |
+| `--tag`        | Set MUD to the latest version with the given tag from npm    | string  |               |
+| `--commit`     | Set MUD to the version based on a given git commit hash from | string  |
```
Comment (alvrs) [inline next-docs/pages/cli/set-version.mdx:23]:
```suggestion
| `--commit`     | Set MUD to the version based on a given git commit hash | string  |
```

**Review by alvrs — APPROVED** [review]

**Pull request merged**